waggle-dance-api/src/test/java/com/hotels/bdp/waggledance/api/model/FederatedMetaStoreTest.java

```diff
@@ -77,4 +79,22 @@ public void toJson() throws Exception {
     String json = mapper.writerFor(FederatedMetaStore.class).writeValueAsString(metaStore);
     assertThat(json, is(expected));
   }
+
+  @Test
+  public void mappedDatabases() {
+    List<String> mappedDatabases = new ArrayList<>();
+    mappedDatabases.add("database");
+    metaStore.setMappedDatabases(mappedDatabases);
+    assertThat(metaStore.getMappedDatabases(), is(mappedDatabases));
+  }
+
+  @Test
+  public void nonEmptyConstructor() {
+    String storeName = "name";
+    String storeUri = "remoteMetaStoreUris";
+    FederatedMetaStore store = new FederatedMetaStore(storeName, storeUri);
```

Comment:
OK, now I see what you were trying to do earlier as these tests extend a base class which be default calls the no-args constructor. I think what you've done now is better than introducing a new test class.

Reply:
Yeah, on Friday I didn't know how to test the other constructor, but I got some help from @teabot to understand how to do it :)